src/services/auth/auth.controller.ts

```diff
@@ -382,17 +385,6 @@ export class AuthController {
       'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c',
     required: true,
   })
-  @ApiBody({
-    schema: {
-      properties: {
-        username: {
-          type: 'string',
-          default: 'test-account',
-          description: 'Username of requested HIVE account',
-        },
-      },
-    },
-  })
```

Comment:
What's wrong with this?

Reply:
Its already done in the dto; this overwrites it

Reply:
Sounds good to me